### PR TITLE
[CAGE-4765] Change Block Group Shape Colors to Uniform Color

### PR DESCRIPTION
**Pull request by ccson**
https://thinkneardev.atlassian.net/browse/CAGE-4765

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -311,165 +307,130 @@
...
-</p>
+  <p id="button" class="button button--blue">
+    <button  class="ui primary button">
+      <a style="color:white;" href="#">VISUALIZE</a>
+    </button>
+  </p>
 
 
 <div class="ui sidebar inverted vertical menu">
-  <p style="color: #fff;">Think Near</p>
```
Comment (ccson) [inline index.html]: This was the only change that did not involve spacing/comments.

File: index.html [hunk 2]
```diff
@@ -35,756 +31,736 @@
...
+    </select>
 
+    <br></br>
 
+    <p>ZIP CODE</p>
 
+    <select id="zip" class="ui multiple search dropdown zip" multiple="">
+    </select>
 
-</div>
```
Comment (ccson) [inline index.html]: This was removed, because there was no matching <div>

File: index.html [hunk 3]
```diff
@@ -32,759 +25,739 @@
...
+          line-opacity: 0.5;\
+        }";
+
+      switch (true) {
+
+        case (joinedSQL.length !== 0):
+          console.log("You selected both!");
+          mapSubLayer2.set({
+            'sql': "SELECT * FROM esri_block_group_shapes_simplified_demo_1 WHERE " + joinedSQL + "",
+            'cartocss': defaultShapeCartocss
```
Comment (ccson) [inline index.html]: The default carto css that will set all shapes to be one color.

File: index.html [hunk 4]
```diff
@@ -4,22 +4,15 @@
...
-
-
-  <!-- <link href="styleguide.css" media="screen" rel="stylesheet" type="text/css" /> -->
   <link href="app.css" media="screen" rel="stylesheet" type="text/css" />
   <link rel="stylesheet" href="https://cartodb-libs.global.ssl.fastly.net/cartodb.js/v3/3.15/themes/css/cartodb.css" />
   <script src="https://cartodb-libs.global.ssl.fastly.net/cartodb.js/v3/3.15/cartodb.js"></script>
 
-  <!--[if lte IE 8]>
-    <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v2/themes/css/cartodb.ie.css" />
-  <![endif]-->
```
Comment (d3cay1) [inline index.html]: This tag will work in IE, we should leave it and remove the below change

File: index.html [hunk 5]
```diff
@@ -32,759 +25,739 @@
...
+              x = response[i]
+            }
           });
+
           response = {
             "success": true,
             "results": cartoResponse.rows
-          },
+          };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: We should remove this `console.log`

File: index.html [hunk 6]
```diff
@@ -32,759 +25,739 @@
...
+
           $.each(cartoResponse.rows, function(row) {
-          var i, x = ""
-          for (i in response) {
-            x = response[i]
-          }
+            var i, x = "";
+            for (i in response) {
+              x = response[i]
```
Comment (d3cay1) [inline index.html]: Why no semi-colon here?

File: index.html [hunk 7]
```diff
@@ -32,759 +25,739 @@
...
-  maxSelections: 5,
-  fields: { name: "name", value: "value" },
-  apiSettings: {
-      cache: false,
-      onResponse: function(cartoResponse) {
+        response = {
+          "success": true,
+          "results": cartoResponse.rows
+        };
+        console.log(response);
```
Comment (d3cay1) [inline index.html]: Remove `console.log`

File: index.html [hunk 8]
```diff
@@ -32,759 +25,739 @@
...
+    $('.ui.multiple.dropdown.state').dropdown({
+      maxSelections: 5,
+      fields: { name: "name", value: "value" },
+      apiSettings: {
+        cache: false,
+        onResponse: function(cartoResponse) {
+        var response = {
+          results: {}
+        };
+        console.log(response);
```
Comment (d3cay1) [inline index.html]: Remove `console.log`

File: index.html [hunk 9]
```diff
@@ -32,759 +25,739 @@
...
           console.log(response);
+
           $.each(cartoResponse.rows, function(row) {
-          var i, x = ""
-          for (i in response) {
-            x = response[i]
-          }
+            var i, x = "";
+            for (i in response) {
+              x = response[i]
```
Comment (d3cay1) [inline index.html]: Semi-colon

File: index.html [hunk 10]
```diff
@@ -32,759 +25,739 @@
...
```
Comment (d3cay1) [inline index.html]: Semi-colon

File: index.html [hunk 11]
```diff
@@ -32,759 +25,739 @@
...
+            }
           });
+
           response = {
             "success": true,
             "results": cartoResponse.rows
-          },
+          };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: Remove `console.log`

File: index.html [hunk 12]
```diff
@@ -32,759 +25,739 @@
...
+    $('.ui.multiple.dropdown.children').dropdown({
+      maxSelections: 5,
+      fields: { name: "name", value: "value" },
+      apiSettings: {
+        cache: false,
+        onResponse: function(cartoResponse) {
           var response = {
             results: {}
           };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: Remove `console.log`

File: index.html [hunk 13]
```diff
@@ -32,759 +25,739 @@
...
+              x = response[i]
+            }
           });
+
           response = {
             "success": true,
             "results": cartoResponse.rows
-          },
+          };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: Remove `console.log`

File: index.html [hunk 14]
```diff
@@ -32,759 +25,739 @@
...
+
           $.each(cartoResponse.rows, function(row) {
-          var i, x = ""
-          for (i in response) {
-            x = response[i]
-          }
+            var i, x = "";
+            for (i in response) {
+              x = response[i]
```
Comment (d3cay1) [inline index.html]: semi-colon

File: index.html [hunk 15]
```diff
@@ -32,759 +25,739 @@
...
+    $('.ui.multiple.dropdown.age').dropdown({
+      maxSelections: 5,
+      fields: { name: "name", value: "value" },
+      apiSettings: {
+        cache: false,
+        onResponse: function(cartoResponse) {
           var response = {
             results: {}
           };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: Remove `console.log`

File: index.html [hunk 16]
```diff
@@ -32,759 +25,739 @@
...
+              x = response[i]
+            }
           });
+
           response = {
             "success": true,
             "results": cartoResponse.rows
-          },
+          };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: Remove `console.log`

File: index.html [hunk 17]
```diff
@@ -32,759 +25,739 @@
...
+
           $.each(cartoResponse.rows, function(row) {
-          var i, x = ""
-          for (i in response) {
-            x = response[i]
-          }
+            var i, x = "";
+            for (i in response) {
+              x = response[i]
```
Comment (d3cay1) [inline index.html]: semi-colon

File: index.html [hunk 18]
```diff
@@ -32,759 +25,739 @@
...
+            }
           });
+
           response = {
             "success": true,
             "results": cartoResponse.rows
-          },
+          };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: remove `console.log`

File: index.html [hunk 19]
```diff
@@ -32,759 +25,739 @@
...
+
           $.each(cartoResponse.rows, function(row) {
-          var i, x = ""
-          for (i in response) {
-            x = response[i]
-          }
+            var i, x = "";
+            for (i in response) {
+              x = response[i]
```
Comment (d3cay1) [inline index.html]: semi-colon

File: index.html [hunk 20]
```diff
@@ -32,759 +25,739 @@
...
+    $('.ui.multiple.dropdown.brand').dropdown({
+      maxSelections: 5,
+      fields: { name: "name", value: "value" },
+      apiSettings: {
+        cache: false,
+        onResponse: function(cartoResponse) {
           var response = {
             results: {}
           };
           console.log(response);
```
Comment (d3cay1) [inline index.html]: remove `console.log`

File: index.html [hunk 21]
```diff
@@ -32,759 +27,698 @@
...
+        } else {
+          var newCartocss = "\
+            #layer {\
+              polygon-fill: ramp([brand_label], (#00AFD7, #E0004D, #FFC72C, #F68D2E, #63666A), (" + brand + "));\
+              opacity: 0.5;\
+            }\
+            #layer::outline {\
+              line-width: 1;\
+              line-color: #FFF;\
+              line-opacity: 0.5;\
```
Comment (d3cay1) [inline index.html]: Please update this to `0.7`

File: index.html [hunk 22]
```diff
@@ -32,759 +27,698 @@
...
+      }";
+
+      function generateCartoCSS(study) {
+        if (study === "none") {
+          return defaultCartocss;
+        } else {
+          var newCartocss = "\
+            #layer {\
+              polygon-fill: ramp([brand_label], (#00AFD7, #E0004D, #FFC72C, #F68D2E, #63666A), (" + brand + "));\
+              opacity: 0.5;\
```
Comment (d3cay1) [inline index.html]: Please update this to `0.7`

**Pull request merged**